pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -118,7 +115,9 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ke *eventingv1alpha1.Kna
 	stages := []func(context.Context, *mf.Manifest, *eventingv1alpha1.KnativeEventing) error{
 		r.ensureFinalizerRemoval,
 		r.install,
-		r.checkDeployments,
+		func(ctx context.Context, manifest *mf.Manifest, ke *eventingv1alpha1.KnativeEventing) error {
+			return common.CheckDeployments(r.kubeClientSet, manifest, &ke.Status)
+		},
```

Comment:
Personally, I'd rather see a one-line member function than a three-line anonymous one. So I'd vote for keeping `r.checkDeployments` and have it do the `common.CheckDeployments` call.

Reply:
+1